dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field_js.jsp

```diff
@@ -328,9 +333,31 @@ var cmsfile=null;
 				return;
 			}
 
+		    <%final Context ctx = VelocityUtil.getWebContext(request, response);%>
+		    <%ctx.put("content", contentlet);%>

```

Comment:
why do you add to the context two variables for the same contentlet?

Reply:
Just for convenience.  If you are trying to script the field, I want peoples script to work first time - either should work.